checkov/arm/parser/cfn_yaml.py

```diff
@@ -6,6 +6,7 @@
 from pathlib import Path
```

Comment:
Perhaps a dumb question but why is this file called `cfn_yaml`? What's the relation to CFN?

Reply:
my best guess is, because it leverages a similar parsing implementation like CFN

Reply:
will create a separate ticket to either rename it or to leverages the existing one.

Reply:
Thanks 🙏

Reply:
adding for visibility #4534